Client/TabManagement/TabManagerImplementation.swift

```diff
@@ -0,0 +1,9 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+// This class subclasses the legacy tab manager temporarily so we can
+// gradually migrate to the new system
+class TabManagerImplementation: LegacyTabManager {}
```

Comment:
So just I understand how this will work, we'll override some method in the new implementation, and use those when a feature flag is true? Or will this work another way?

Reply:
Yes, we'll override and then in the function use the flag to either call the super version of the function or do the new stuff instead.